tests/fork/config-wait-exec.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Mystikos configuration version number
+    "version": "0.1",
+
+    // OpenEnclave specific values
+    "Debug": 1,
+    "StackMemSize": "256k",
+    "NumUserThreads": 2,
+    "ProductID": 1,
+    "SecurityVersion": 1,
+
+    // Mystikos specific values
+    "MemorySize": "40m",
+    "ApplicationPath": "/bin/hello",
+    "HostApplicationParameters": true,
+    "ForkMode":"pseudo_wait_for_exec"
```

Comment:
The strings here don't match what is in ``myst_fork_mode_t`` (I saw ``myst_fork_pseudo_wait_for_exit_exec`` but not ``myst_fork_pseudo_wait_for_exec``).

Reply:
renamed